src/datasets/commands/dummy_data.py

```diff
@@ -124,33 +124,46 @@ def _create_dummy_data(
                 with open(src_path, "r", encoding=encoding) as src_file:
```

Comment:
Can you remove these changes ?

If you feel like they can be useful, please open another PR

Reply:
Yes, sorry, it was not intended to add this in the same PR, I was waiting to add it another one, and it lacks some check on the Exception. I must have done a "commit -am" instead of a "commit -m" somewhere. It had been a long long time to add all the dummy_data files, I must have lowered the guard at some point...